src/datatypes/character/Character.cpp

```diff
@@ -0,0 +1,81 @@
+//
+// Created by Carolin on 02.04.2020.
+//
+
+#include "Character.hpp"
+
+namespace spy::character {
+
+    const spy::util::UUID &spy::character::Character::getCharacterId() const {
```

Comment:
Hier könnte man für ein bisschen lesbareren code den namespace weglassen, da man sich ja schon im passenden namespace befindet (auch bei den anderen funktionen):
```suggestion
    const spy::util::UUID &Character::getCharacterId() const {
```

Reply:
done: befecea